configure.ac

```diff
@@ -121,7 +121,7 @@ AM_COND_IF([DEPENDENCY_CHECKS], [
                                   [AC_MSG_FAILURE([--with-harfbuzz was given, but harfbuzz not found])]) ])
 
     AM_COND_IF([FONT_VARIATIONS],
-               [PKG_CHECK_MODULES(FONT_VARIATIONS, [harfbuzz-subset >= 6.0.0],
+               [PKG_CHECK_MODULES(HARFBUZZ_SUBSET, [harfbuzz-subset >= 6.0.0],
```

Comment:
Ha sorry about this. I have it fixed locally in a more expansive rebase branch but forgot to stuff the fixup here for visibility.

Reply:
No problem. But I wonder why CI is still running and failing the variations test when they are not being built with variations support.